### PR TITLE
Add: Shortcuts for some actions

### PR DESCRIPTION
**Pull request by 1kuemmel1**
- Add: Shortcuts for civil orders.

This simply add some key shortcuts for the actions:
civil oder: stop 
civil oder: get down
civil oder: go away

see: configure>> controls>> configure addons >> Hearts and Minds: Mission (drop down menu)

Final test :
- [x] local
- [x] server

### CODE REVIEW

**Comment by 1kuemmel1** [comment]
Sorry for my mistake.
i hope it is correct right now.

How did i add lables?

**Comment by Vdauphin** [comment]
Don't worry your not the first who did that mistake. It is okay now. 

Only autorized people can put lable. I just labled your PR to the next major update. I will take a look as soon as possible.

Thank for you!

File: =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/int/shortcuts.sqf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#include "\a3\editor_f\Data\Scripts\dikCodes.h"
+#define BTC_PLAY_FBSOUND true    //set false if you do not want a "key-pressed-feedback" (sound)
+#define BTC_FBSOUND "ClickSoft"  //really quiet sound
+
+//Order Stop
+["Hearts and Minds: Mission", "btc_HaM_Action_civStop", ["Civil Order: Stop", "Order a civilian to stop"],{
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/int/shortcuts.sqf]: Can you advise player in https://github.com/Vdauphin/HeartsAndMinds/blob/master_daily/=BTC=co@30_Hearts_and_Minds.Altis/core/doc.sqf#L123-L133 file about this new shortcut ?

like : 
```
Any player can give orders to civilians.<br/>
<img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\interact_ca.paa' width='20' height='20'/> Open your self interaction menu and select 'ORDERS' or use shortcut available under: configure>> controls>> configure addons >> Hearts and Minds: Mission (drop down menu). <br/>

```
for exemple?

Comment (1kuemmel1) [reply =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/int/shortcuts.sqf]: Sure, i will take care of these.

**Comment by 1kuemmel1** [comment]
Sorry. I think I did something wrong. See under files changed.

**Comment by Vdauphin** [comment]
Hello,
For me it is perfect now.
What is wrong ?

**Comment by 1kuemmel1** [comment]
Files changed 9 , but i just changed 4 files. I did´t want to make something wrong.

**Comment by Vdauphin** [comment]
Don't worry, it is okay, those change are FIX from the master_stable

**Comment by Vdauphin** [comment]
In https://github.com/Vdauphin/HeartsAndMinds/wiki/InGame-documentation/2cc934129323f12568713b4c8629fef44fd1619c

**Pull request merged**